payments-core/src/main/java/com/stripe/android/payments/core/authentication/DefaultPaymentAuthenticatorRegistry.kt

```diff
@@ -34,34 +31,23 @@ private typealias Authenticator = @JvmSuppressWildcards PaymentAuthenticator<Str
 internal class DefaultPaymentAuthenticatorRegistry @Inject internal constructor(
     private val noOpIntentAuthenticator: NoOpIntentAuthenticator,
     private val sourceAuthenticator: SourceAuthenticator,
-    @IntentAuthenticatorMap private val paymentAuthenticators: Map<AuthenticatorKey, Authenticator>
+    @IntentAuthenticatorMap private val paymentAuthenticators: Map<AuthenticatorKey, Authenticator>,
+    @Named("INCLUDE_PAYMENT_SHEET_AUTHENTICATORS") private val includePaymentSheetAuthenticators: Boolean,
```

Comment:
Nit: Should either have this as a constant or use an enum like `AuthenticationContext` or `Surface` or something like that.